content/source/docs/enterprise-beta/api/policies.html.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: enterprise2
+page_title: "Policies - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-policy"
+---
+
+# Policies API
+
+-> **Note**: These API endpoints are in beta and may be subject to change.
+
+[Sentinel Policy as Code](/docs/enterprise-beta/sentinel/index.html) is an embedded policy as code framework integrated with Terraform Enterprise. Policies are defined in organizations and enforced on all workspace Runs between a plan and an apply. This doc covers operations to create, read, update, and delete the Sentinel Policies in an organization. The [Runs API](/docs/enterprise-beta/api/run.html) covers the steps for reading and overriding policy checks in a workspace run.
```

Comment:
Policies are defined in organizations and enforced on all workspace Runs between a plan and an apply. A plan must pass the policy in order to proceed to the apply step.